src/acvp_transport.c

```diff
@@ -756,8 +756,8 @@ ACVP_RESULT acvp_transport_get(ACVP_CTX *ctx,
 #endif
 }
 
-ACVP_RESULT acvp_get_request_status (ACVP_CTX *ctx, 
-                                     const char *url) {
+ACVP_RESULT acvp_get(ACVP_CTX *ctx, 
```

Comment:
Same thing, can this function be removed?

Reply:
done